.ci/setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Setup workflow:
+# 1. setup environment (install qemu, kernel and image)
+# 2. Clone repos, fetch branches, build and install
+
+set -e
+
+export TESTS_CI_DIR=${GOPATH}/src/github.com/clearcontainers/tests/.ci
+
+echo "Set up environment"
+bash -f ${TESTS_CI_DIR}/setup_env_ubuntu.sh
```

Comment:
This is safe for now, but I wonder if we should add a check at some point to ensure the host system is actually running Ubuntu (`grep -q "^ID=ubuntu$" /etc/os-release` or similar).